### PR TITLE
build(python): remove python 3.6 support

### PR DESCRIPTION
**Pull request by Linkid**
Python 3.6 reaches end of life in December 2021.

This PR fixes CI too.

### CODE REVIEW

**Comment by hramezani** [comment]
Thanks @Linkid for the patch.
As Django 2.2 will be supported until April 2022, I would suggest keeping Python 3.6 and remove it with Django 2.2

**Comment by Linkid** [comment]
@hramezani: I see what you mean, but Python 3.6 is not supported anymore since December 2021. And even if Django 2.2 supports it, it is like Python 3.5 which is not supported anymore but Django 2.2 supports it officially.

File: tox.ini [hunk 1]
```diff
@@ -29,11 +29,12 @@ basepython =
     py39: python3.9
     py38: python3.8
     py37: python3.7
-    py36: python3.6
 
 [testenv:py38-docs]
 changedir = docs/source
-deps = -rdocs/requirements.txt
+deps =
+    Django>=3.2,<3.3
```
Comment (hramezani) [inline tox.ini:36]: Why you didn't add this to the `docs/requirements.txt`?

Comment (Linkid) [reply tox.ini:36]: Seems like all the auditlog package is needed (and installed with tox by default).

File: docs/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # Docs requirements
+.
```
Comment (hramezani) [inline docs/requirements.txt:2]: I think we don't need this line.

**Review by hramezani — APPROVED** [review]

**Comment by Linkid** [comment]
Thanks :)

**Pull request merged**